src/main/java/net/atos/zac/app/util/LocalDateReader.java

```diff
@@ -18,7 +18,7 @@
 @Produces(MediaType.TEXT_PLAIN)
 public class LocalDateReader implements MessageBodyReader<LocalDate> {
 
-    DateTimeFormatter DEFAULT_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'hh:mm+01:00");
```

Comment:
Weird code... good that you caught this. Just for my understanding: this has nothing to do with the rest of the PR right?

Reply:
While testing manually the upload breaks here on my machine.

So had to fix this to allow other time-zones